test/e2e/common/setup-kubestellar.sh

```diff
@@ -109,14 +104,25 @@ esac
 : Install the core-chart
 :
 
-pushd "${SRC_DIR}/../../.."
-make kind-load-image
-make install-local-core-chart \
-  INSTALL_KUBEFLEX=false \
-  ITS_NAME=its1 \
-  DEFAULT_WDS_NAME=wds1 \
-  KUBESTELLAR_CONTROLLER_MANAGER_VERBOSITY=$KUBESTELLAR_CONTROLLER_MANAGER_VERBOSITY
-popd
+if [ "$use_release" = true ] ; then
+  kubestellar_latest_release=$(curl -sL https://github.com/kubestellar/kubestellar/releases/latest | grep "</h1>" | tail -n 1 | sed -e 's/<[^>]*>//g' | xargs)
+  helm upgrade --install ks-core oci://ghcr.io/kubestellar/kubestellar/core-chart \
+    --version ${kubestellar_latest_release//v} \
```

Comment:
That's a fragile way to strip off the leading "v" --- it actually deletes every "v". There is a better way.

```console
bash-3.2$ release=v0.24.1-preview.1

bash-3.2$ echo ${release//v}
0.24.1-preiew.1

bash-3.2$ echo ${release#v}
0.24.1-preview.1
```

Reply:
ok